lgc/patch/ShaderInputs.cpp

```diff
@@ -333,46 +333,53 @@ void ShaderInputs::fixupUses(Module &module, PipelineState *pipelineState) {
     ShaderInputsUsage *inputsUsage = getShaderInputsUsage(stage);
     for (unsigned kind = 0; kind != static_cast<unsigned>(ShaderInput::Count); ++kind) {
       ShaderInputUsage *inputUsage = inputsUsage->inputs[kind].get();
-      if (inputUsage && inputUsage->entryArgIdx != 0) {
-        Argument *arg = getFunctionArgument(&func, inputUsage->entryArgIdx);
-        arg->setName(getInputName(static_cast<ShaderInput>(kind)));
-        for (Instruction *&call : inputUsage->users) {
-          if (call && call->getFunction() == &func) {
-            call->replaceAllUsesWith(arg);
-            call->eraseFromParent();
-            call = nullptr;
-          }
+      if (!inputUsage)
+        continue;
+      Value *value = nullptr;
+      {
```

Comment:
What is the benefit of putting this in a separate scope?

Reply:
It is going to serve an internal change.